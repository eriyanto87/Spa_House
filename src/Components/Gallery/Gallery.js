import React from "react";
import img1 from "../../images/massage-sushi3.jpg";
import img2 from "../../images/home-massage-3.png";
import img3 from "../../images/home-massage-dtla.png";
import img4 from "../../images/massage-mothersday2.jpg";
import img5 from "../../images/massage-office.jpg";
import img6 from "../../images/massage-sushi2.jpg";
import img7 from "../../images/home-massage2.png";
import img8 from "../../images/home-massageedit.png";
import img9 from "../../images/sushi-bday.png";
import img10 from "../../images/massage-sushi1.jpg";
import Nav from "../Nav/Nav";
import Footer from "../Footer/Footer";
import Header from "../Header/Header";

export default function Gallery() {
  return (
    <div className="gallery">
      <Nav />
      <Header />
      <img src={img1} alt="chair massage at a dog birthday party" />
      <img src={img2} alt="massage at home" />
      <img src={img3} alt="massage with dtla view" />
      <img src={img4} alt="chair massage for a mother's day gift" />
      <img src={img5} alt="chair massage at an office" />
      <img src={img6} alt="chair massage at a dog birthday party" />
      <img src={img7} alt="massage at home" />
      <img src={img8} alt="massage at home" />
      <img src={img9} alt="sushi birthday" />
      <img src={img10} alt="massage at sushi birthday" />
      <Footer />
    </div>
  );
}
