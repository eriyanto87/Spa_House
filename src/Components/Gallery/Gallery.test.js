import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import Gallery from "./Gallery";

describe("Gallery component", () => {
  it("Gallery Snapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Gallery />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <Gallery />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
});
