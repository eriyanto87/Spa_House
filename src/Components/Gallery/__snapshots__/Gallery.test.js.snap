// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery component Gallery Snapshot 1`] = `
<div
  className="gallery"
>
  <div
    className="nav"
  >
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Home 
        </a>
      </li>
      <li>
        <a
          href="/about"
          onClick={[Function]}
        >
          About 
        </a>
      </li>
      <li>
        <a
          href="/treatments"
          onClick={[Function]}
        >
          Treatments 
        </a>
      </li>
      <li>
        <a
          href="/pricing"
          onClick={[Function]}
        >
          Pricing 
        </a>
      </li>
      <li>
        <a
          href="/contact"
          onClick={[Function]}
        >
          Contact Us 
        </a>
      </li>
      <li>
        <a
          href="/gallery"
          onClick={[Function]}
        >
          Gallery
        </a>
      </li>
    </ul>
  </div>
  <div
    className="sm-text"
  >
    <h1>
      SPA HOUSE
    </h1>
  </div>
  <img
    alt="chair massage at a dog birthday party"
    src="massage-sushi3.jpg"
  />
  <img
    alt="massage at home"
    src="home-massage-3.png"
  />
  <img
    alt="massage with dtla view"
    src="home-massage-dtla.png"
  />
  <img
    alt="chair massage for a mother's day gift"
    src="massage-mothersday2.jpg"
  />
  <img
    alt="chair massage at an office"
    src="massage-office.jpg"
  />
  <img
    alt="chair massage at a dog birthday party"
    src="massage-sushi2.jpg"
  />
  <img
    alt="massage at home"
    src="home-massage2.png"
  />
  <img
    alt="massage at home"
    src="home-massageedit.png"
  />
  <img
    alt="sushi birthday"
    src="sushi-bday.png"
  />
  <img
    alt="massage at sushi birthday"
    src="massage-sushi1.jpg"
  />
  <div>
    <p>
      Â© 2020
    </p>
  </div>
</div>
`;
